Samantha-samanthalpark/Homework-3/object-homework/app.js

```diff
@@ -0,0 +1,94 @@
+/////////////////////////////////////////////////////////////
+
+///////////////////////// Challenge /////////////////////////
+
+/////////////////////////////////////////////////////////////
+
+// Write your answers inside this file, at the places where it's indicated by the comments.
+
+
+// 1. Suppose that we wanted to create a browser-based calendar program for keeping track of things. In comments, list at least three reasonable abstractions that you might use to build this program; for each abstraction, list out at least two properties and behaviors that it would make sense for that abstraction to have. Below is an example - please write your answer in the same format.
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+// 'Calendar'
+// Description: Manage/keep track of events
+// Every 'event' has
+// - a day
+// - a month
+// - a year
+// - day of the week
+// - name
+// - location
+// - duration
+// Every 'calendar' can
+// - add event
+// - edit event
+// - move event
+// - delete event
+
+// Answer Ends Here
+
+
+// 2. Create an Object literal that lines up with the following description. Store it in the variable 'pet_owner', below. Be sure to give it reasonable values for each of its properties.
+
+  // 'Owner'
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'owner' has:
+  // - a name
+  // - an address
+
+
+
+// Answer Starts Here
+let pet_owner = {
+  name = "first name last name",
```

Comment:
Remember that to declare a property in an object, you use a colon ( : ) rather than an equal sign between the key and value. So this line, for instance, would read `name: "first name last name",`.